core/arch/arm/plat-vexpress/conf.mk

```diff
@@ -115,8 +115,13 @@ endif
 ifeq ($(PLATFORM_FLAVOR),qemu_armv8a)
 CFG_TEE_CORE_NB_CORE = 4
 # [0e00.0000 0e0f.ffff] is reserved to early boot
+ifeq ($(CFG_CORE_SEL2_SPMC),y)
+CFG_TZDRAM_START ?= 0x0e704000
```

Comment:
How was this address determined?

Reply:
This address is calculated based on following configuration of the Secured DRAM in the trusted firmware,

Hafnium (BL32)
Load address - 0xe100000  Size = 0x600000

OP-TEE
Load address - 0xe700000  Size = 0x900000
entrypoint-offset = <0x4000

However from the testing, the memory required for the  hafnium could be reduce to 0x200000 and 
OP-TEE  load address could be  0xe300000.